packages/adblocker/src/resources.ts

```diff
@@ -21,13 +23,44 @@ function btoaPolyfill(buffer: string): string {
 }
 
 export interface Resource {
+  names: string[];
+  body: string;
   contentType: string;
+}
+
+interface Scriptlet {
+  names: string[];
   body: string;
+  dependencies: string[];
+  executionWorld: Scripting.ExecutionWorld;
+  requiresTrust: boolean;
+}
+
+export interface ResourcesDistribution {
+  redirects: Array<{
+    names: string[];
+    content: string;
+    contentType: string;
+    encoding?: 'base64';
+  }>;
+  scriptlets: Array<{
+    names: string[];
+    content: string;
+    dependencies: string[];
+    executionWorld?: Scripting.ExecutionWorld;
+    requiresTrust?: boolean;
+  }>;
 }
 
-// TODO - support # alias
 // TODO - support empty resource body
 
+export const wrapScriptletBody = (script: string, dependencies: string[] = []): string =>
```

Comment:
that was originally exported to be used in tests, but after test cleanup it is no longer used in testing. 
Maybe we can make it private and migrate its tests to `getScriptlet`?